gtfs/tests/test_filtering.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from model_bakery import baker
+
+from gtfs.models import Feed, Route, Stop
+from gtfs.tests.utils import get_feed_for_maas_operator
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("has_permission", [True, False])
+def test_filtering_feeds_for_maas_operators(maas_operator, has_permission):
+    feed = get_feed_for_maas_operator(maas_operator, has_permission)
+
+    feeds = Feed.objects.for_maas_operator(maas_operator)
+
+    if has_permission:
+        assert feed in feeds
+    else:
+        assert feed not in feeds
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("has_permission", [True, False])
+def test_filtering_routes_for_maas_operator(maas_operator, has_permission):
+    feed = get_feed_for_maas_operator(maas_operator, has_permission)
+    route = baker.make(Route, feed=feed)
+
+    routes = Route.objects.for_maas_operator(maas_operator)
+
+    if has_permission:
+        assert route in routes
+    else:
+        assert route not in routes
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("has_permission", [True, False])
+def test_filtering_stops_for_maas_operator(maas_operator, has_permission):
+    feed = get_feed_for_maas_operator(maas_operator, has_permission)
+    stop = baker.make(Stop, feed=feed)
+
+    stops = Stop.objects.for_maas_operator(maas_operator)
+
+    if has_permission:
+        assert stop in stops
+    else:
+        assert stop not in stops
```

Comment:
I think an easy way to make these tests even more comprehensive could be to add a second feed/route/stop with a different maas operator, so that the test also makes sure having the permission applies to the exact operator's objects only.